modules/openy_gc_auth/modules/openy_gc_auth_reclique_sso/openy_gc_auth_reclique_sso.services.yml

```diff
@@ -1,6 +1,4 @@
 services:
   openy_gc_auth.reclique_sso_client:
     class: Drupal\openy_gc_auth_reclique_sso\SSOClient
-    arguments: [ '@logger.factory', '@config.factory',
-                 '@http_client', '@csrf_token',
-                 '@user.private_tempstore' ]
+    arguments: ['@logger.factory', '@config.factory', '@http_client', '@csrf_token', '@tempstore.private' ]
```

Comment:
I see that you've updated the arguments. It looks like we have to update constructor on the SSoClient.php side.
Isnt it?

Reply:
No, there was already the core class used

Reply:
https://github.com/fivejars/openy_gated_content/blob/69ac8e0d8b98f6f2be5d73bec06a1b68fbf06d7c/modules/openy_gc_auth/modules/openy_gc_auth_reclique/src/Form/VirtualYReCliqueLoginForm.php#L99

You replaced `PrivateTempStoreFactory` by `PrivateTempStore` on 69 line, but constructor contains `PrivateTempStoreFactory`. @NightHunterSV Are you sure this is correct?

Reply:
Yes, because there is a code in the constructor:
https://github.com/fivejars/openy_gated_content/blob/90a8feb4cb3e5ec781e7d4d771d4e5d8c05cdd59/modules/openy_gc_auth/modules/openy_gc_auth_yusa/src/Form/VirtualYUSALoginForm.php#L121
So the `$private_temp_store` should be an instance of `PrivateTempStoreFactory`, and it's `->get` returns an object of `PrivateTempStore`